2019-11-06 08:03:10.897 -05:00 [Warning] Ingreso
2019-11-06 08:23:26.011 -05:00 [Error] String '06' was not recognized as a valid DateTime.
2019-11-06 08:26:17.897 -05:00 [Warning] Ingreso
2019-11-06 08:30:28.915 -05:00 [Warning] Ingreso
2019-11-06 08:39:26.528 -05:00 [Warning] Ingreso
2019-11-06 08:39:45.709 -05:00 [Error] The parameterized query '(@startdate datetime,@enddate datetime,@subsidiaryId int,@numord' expects the parameter '@numorder', which was not supplied.
2019-11-06 08:39:54.598 -05:00 [Error] An unhandled exception has occurred while executing the request.
System.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@startdate datetime,@enddate datetime,@subsidiaryId int,@numord' expects the parameter '@numorder', which was not supplied.
   at EKT.FACT.Service.Controllers.ReceiptController.GetReceiptManager(String startdate, String enddate, Int32 subsidiaryId, String numorder, String documentNumber) in D:\ALINH\OPER\BAZ\FACTEKT\Fuentes\EKT.FACT\EKT.FACT.Service\Controllers\ReceiptController.cs:line 58
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
ClientConnectionId:2b8261e8-7ca9-4863-a213-3eff24f3d74e
Error Number:8178,State:1,Class:16
